### PR TITLE
Update styling for program block editor.

### PR DESCRIPTION
**Pull request by AWBaumann**
### Description
Added styling to the main admin page.

Note: The SVG usage is ugly as sin. I'm reasonably sure we'll be doing things differently, but it's nice to get some placeholder images in now.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

image

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/admin/AdminLayout.java [hunk 1]
```diff
@@ -36,14 +35,13 @@ public Content render(DomContent... mainDomContents) {
...
-                Styles.BORDER_GRAY_200,
-                Styles.PX_2,
-                Styles.SHADOW_LG);
+    mainContent.withClasses(
+        Styles.BG_WHITE,
+        Styles.BORDER,
+        Styles.BORDER_GRAY_200,
+        Styles.PX_2,
+        Styles.SHADOW_LG,
+        StyleUtils.joinStyles(mainStyles));
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/admin/AdminLayout.java:44]: ahhhh

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java [hunk 2]
```diff
@@ -19,14 +20,24 @@
...
 
 public class ProgramBlockEditView extends BaseHtmlView {
 
   private final AdminLayout layout;
 
+  private static final String CREATE_BLOCK_FORM = "block-create";
+  private static final String DELETE_BLOCK_FORM = "block-delete";
+  private static final String UPDATE_BLOCK_FORM = "block-update";
+  private static final String ADD_QUESTION_FORM = "question-add";
+  private static final String DELETE_QUESTION_FORM = "question-delete";
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:38]: Suggestion (non-blocking): add `_ID` to the end of the constant name?

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java [hunk 3]
```diff
@@ -19,14 +20,24 @@
...
+  private static final String POST = "post";
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:39]: Suggestion (blocking): please move this somewhere more reusable

File: universal-application-tool-0.0.1/app/views/components/Icons.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        .attr("fill", "currentColor")
+        .attr("stroke", "currentColor")
+        .attr("stroke-width", "1%")
+        .attr("aria-hidden", "true")
+        .with(
+            new ContainerTag("path")
+                .attr("xmlns", "http://www.w3.org/2000/svg")
+                .attr("d", pathString));
+  }
+}
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/components/Icons.java:67]: praise: in the PR description you said this was ugly as sin but tbh it looks fairly similar to how several different React project I've seen handled SVGs.

File: universal-application-tool-0.0.1/app/views/components/Icons.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          + "  c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z"
+          + " M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17 "
+          + " s-17-7.626-17-17S14.61,6,23.984,6z";
+  public static final String TEXT_SVG_PATH =
+      "M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0"
+          + " 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4"
+          + " 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5"
+          + " 0-2.21-1.79-4-4-4z";
+
+  public static ContainerTag questionTypeSvg(QuestionType type, int size) {
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/components/Icons.java:27]: Suggestion (blocking): whenever we specify size please make the units clear. Two options for that: one would be to go TurboJava and have a class that represents a magnitude in a given unit (this is probably the safest/best option):

```java
@AutoValue
class PixelUnit {
  abstract int value;
}

@AutoValue
class PercentUnit {
  abstract int value;
}
```

or name the param to suggest the unit the `int` will be used for e.g. `sizePixels` or `sizePercent` which is less safe but quicker for now.

**Review by bion — APPROVED** [review]

File: universal-application-tool-0.0.1/app/views/components/Icons.java (hunk 5 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/components/Icons.java:27]: If you'd prefer to do the latter for now please file an issue for doing the former unless you'd like to discuss it more.

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java (hunk 2 above)
Comment (AWBaumann) [reply universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:38]: Done.

File: universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java (hunk 3 above)
Comment (AWBaumann) [reply universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java:39]: Done. Not sure exactly where this belongs, but ViewUtils seems to work for now.

File: universal-application-tool-0.0.1/app/views/components/Icons.java (hunk 5 above)
Comment (AWBaumann) [reply universal-application-tool-0.0.1/app/views/components/Icons.java:27]: Done.

**Pull request merged**